MagaluCloud/magalu
# fix(openapi): remove name and id limit from VM instances list

## Description

This PR changes the output of `go run . virtual-machine instances list` to display all columns.

### Pull request checklist


## How to test it

Run `go run . virtual-machine instances list` and check that it displays the full table output. The output should no longer contain only the `NAME` and `ID` columns.
